tests/conftest.py

```diff
@@ -2625,6 +2625,7 @@ def ceph_toolbox(request):
     """
     deploy = config.RUN['cli_params']['deploy']
     teardown = config.RUN['cli_params'].get('teardown')
-    if not deploy and not teardown:
+    skip_ocs = config.ENV_DATA['skip_ocs_deployment']
+    if not (deploy or teardown or skip_ocs):
```

Comment:
Just thinking, if we do deploy of ocp with skip ocs deploy I guess we have condition satisfied with is not deployment, why do we need this second one?

Reply:
Once install stage is completed and As part test stage --deploy is not passed